dali/imgcodec/decoders/libjpeg_turbo.cc

```diff
@@ -21,10 +21,12 @@
 namespace dali {
 namespace imgcodec {
 
-DecodeResult LibJpegTurboDecoderInstance::Decode(SampleView<CPUBackend> out,
```

Comment:
This decoder was not parallelized due to implementing Decode instead of DecodeImplTask